### PR TITLE
[fix] fix `HOME environment variable not found` on Windows platform

### PR DESCRIPTION
**Pull request by djy0**
## Motivation

trying to fix https://github.com/starcoinorg/starcoin/issues/3400

### Have you read the [Contributing Guidelines on pull requests](https://github.com/move-language/move/blob/main/CONTRIBUTING.md#pull-requests)?

yes

### CODE REVIEW

File: language/testing-infra/transactional-test-runner/src/framework.rs [hunk 1]
```diff
@@ -739,7 +739,10 @@ fn handle_expected_output(test_path: &Path, output: impl AsRef<str>) -> Result<(
     if !exp_path.exists() {
         std::fs::write(&exp_path, "").unwrap();
     }
-    let expected_output = std::fs::read_to_string(&exp_path).unwrap();
+    let expected_output = std::fs::read_to_string(&exp_path)
+        .unwrap()
+        .replace("\r\n", "\n")
+        .replace("\r", "\n");
```
Comment (jolestar) [inline language/testing-infra/transactional-test-runner/src/framework.rs:745]: this change has been included in #132

Comment (djy0) [reply language/testing-infra/transactional-test-runner/src/framework.rs]: thanks, this was removed from current pr

**Review by wrwg — APPROVED** [review]

**Comment by jolestar** [comment]
@djy0 please help to fix the test

**Review by jolestar — APPROVED** [review]

**Comment by djy0** [comment]
https://github.com/move-language/move/blob/1a0214a1aa850830fb993ca86746d01132135109/CONTRIBUTING.md?plain=1#L31

Sorry, how to `complete the Contributor License Agreement (CLA)` ?

**Comment by wrwg** [comment]
> https://github.com/move-language/move/blob/1a0214a1aa850830fb993ca86746d01132135109/CONTRIBUTING.md?plain=1#L31
> 
> Sorry, can't find out how to `complete the Contributor License Agreement (CLA)`

That's bogus and not longer in effect.

**Pull request merged**